src/proto/semantifly.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+package semantifly;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "accretional.com/semantifly/proto";
+
+service Semantifly {
+  rpc Add(AddRequest) returns (AddResponse) {}
+  rpc Delete(DeleteRequest) returns (DeleteResponse) {}
+  rpc Get(GetRequest) returns (GetResponse) {}
+  rpc Update(UpdateRequest) returns (UpdateResponse) {}
+  rpc LexicalSearch(LexicalSearchRequest) returns (LexicalSearchResponse) {}
+}
+
+message AddRequest {
+  DataType data_type = 1;
+  SourceType source_type = 2;
+  bool make_copy = 3;
+  repeated string data_uris = 4;
+}
+
+message AddResponse {
+  string message = 1;
+}
+
+message DeleteRequest {
+  bool delete_copy = 1;
+  repeated string data_uris = 2;
+}
+
+message DeleteResponse {
+  string message = 1;
+}
+
+message GetRequest {
+  string name = 1;
+}
+
+message GetResponse {
+  string content = 1;
+  string message = 2;
+}
+
+message UpdateRequest {
+  string name = 1;
+  DataType data_type = 2;
+  SourceType source_type = 3;
+  bool update_copy = 4;
+  string data_uri = 5;
+}
+
+message UpdateResponse {
+  string message = 1;
```

Comment:
What kind of message? Error message probably right? So maybe call this (and other "messages" used for the same thing) error_message

Reply:
message stores all messages in the console (e.g. output messages, error messages that still allow the function to continue, etc.) not necessarily only error messages

Reply:
We should not output any non-error/warning messages in this response. Please only include warnings and actual errors and change this to error_message

Reply:
done

Reply:
fixed